socorro/tests/processor/rules/test_mozilla.py

```diff
@@ -1876,16 +1879,6 @@ def test_junk_data(self, tmp_path, os_name, os_version, expected):
         rule.act(raw_crash, dumps, processed_crash, str(tmp_path), status)
         assert processed_crash["os_pretty_version"] == expected
 
-    def test_dotdict(self, tmp_path):
```

Comment:
This is a vestige of the past when we had `DotDict` and the raw and processed crash structures were `DotDict` structures sometimes except on days where sunrise is on the 00 of the hour when they're dicts. Those days are so over so we can remove this.

It probably should have been removed in d727e09e4c5d4f0f7c818cc7c164a818c38e7e0c.